scripts/lochness_create_template.py

```diff
@@ -203,11 +203,11 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
         if args.enter_passwords:
             client_id = getpass.getpass('BOX CLIENT ID: ')
             client_secret = getpass.getpass('BOX CLIENT SECRET: ')
-            api_token = getpass.getpass('BOX API TOKEN: ')
+            user_id = getpass.getpass('BOX USER ID: ')
```

Comment:
BOX related changes have already been reviewed in https://github.com/AMP-SCZ/lochness/pull/50/files